.github/workflows/update-grype-release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
       - run: |
           LATEST_VERSION=$(curl "https://api.github.com/repos/anchore/grype/releases/latest" 2>/dev/null | jq -r '.tag_name')
-          echo "export const VERSION = \"$LATEST_VERSION\";" > GrypeVersion.js
+          echo "exports.GRYPE_VERSION = \"$LATEST_VERSION\";" > GrypeVersion.js
```

Comment:
This is hopefully the last thing I botched porting the `sbom-action` auto-update to this repo. I noticed this because it should have failed but still didn't appear to have run the precommit hooks, which prompted updating husky to the same pattern.